content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,301 +1,199 @@
 
-```terminal
-curl -v https://UNIQUE_IDENTIFIER.execute-api.us-east-1.amazonaws.com/dev/seed
+```code no-copy
+Serverless: Copy prisma schema for app...
+Serverless: Generate prisma client for app...
+Serverless: Remove unused prisma engine:
+Serverless: - node_modules/.prisma/client/libquery_engine-darwin.dylib.node
+Serverless: - node_modules/@prisma/engines/libquery_engine-darwin.dylib.node
+Serverless: - node_modules/@prisma/engines/migration-engine-darwin
+Serverless: - node_modules/@prisma/engines/prisma-fmt-darwin
+Serverless: Copying existing artifacts...
 ```
 
-<Admonition type = "info">
+### Binary targets in <inlinecode>schema.prisma</inlinecode>
 
-The `serverless.yml` configuration file contains the endpoint and function configuration. You can update this file to add or change endpoints. For more AWS specific configuration information, see the AWS provider configuration in the [Serverless Framework documentation](https://serverless.com/framework/docs/providers/aws/guide/functions/).
+The Prisma schema should contain the following in the `generator` block:
 
-</Admonition >
+```prisma
+binaryTargets = ["native", "rhel-openssl-1.0.x"]
+```
 
-## Deploy only the required files
+This is necessary because the local runtime is different from the Lambda runtime. Add the [`binaryTarget`](/reference/api-reference/prisma-schema-reference#binarytargets-options) to make the compatible Prisma engine binary available.
 
-To reduce your deployment footprint, you can change your deployment process so that you only upload the files that your application needs. The method for doing so depends on whether you use the Serverless Framework or not.
+#### Lambda functions with arm64 architectures
 
-### If you use the Serverless Framework
+Lambda functions that use [arm64 architectures (AWS Graviton2 processor)](https://docs.aws.amazon.com/lambda/latest/dg/foundation-arch.html#foundation-arch-adv) must use an `arm64` precompiled binary.
 
-You can use the Serverless configuration file to [deploy only the binary that your project needs](/guides/deployment/deployment-guides/use-prisma-with-serverless-framework#reduce-the-package-footprint).
+In the `generator` block of your `schema.prisma` file, add the following:
 
-### If you do not use the Serverless Framework
+```prisma file=schema.prisma
+binaryTargets = ["native", "linux-arm64-openssl-1.0.x"]
+```
 
-If you do not use the Serverless Framework to deploy your project, you can use the following method to remove any unneeded files.
+Update the Serverless configuration file to package the `arm64` binary, as follows:
 
-1. Create a zip script that only zips the files that your project needs. [Example script](https://github.com/prisma/ecosystem-tests/blob/dev/platforms-serverless/lambda/zip.sh).
+```code file=serverless.yml
+package:
+  patterns:
+    - '!node_modules/.prisma/client/libquery_engine-*'
+    - 'node_modules/.prisma/client/libquery_engine-linux-arm64-*'
```

Comment:
Hmmm, is this also supported by the plugin or would that exclude that file?
Something to check and potentially send a PR for if it is not supported.